libraries/vue/package.json

```diff
@@ -14,11 +14,11 @@
     "karma-firefox-launcher": "2.1.2",
     "karma-sourcemap-loader": "0.3.8",
     "karma-webpack": "5.0.0",
-    "vue-template-compiler": "2.6.14",
+    "@vue/compiler-sfc": "^3.1.0",
     "webpack": "5.65.0"
   },
   "dependencies": {
     "@webcomponents/webcomponentsjs": "2.6.0",
-    "vue": "2.6.14"
+    "vue": "^3.1.0"
```

Comment:
Nit: Bump minor version to 3.2.30

Reply:
the '^' operator takes care of that, and the package-lock.json actually specifies the version that we test with